ts/screens/wallet/ManuallyIdentifyTransactionScreen.tsx

```diff
@@ -0,0 +1,121 @@
+import { none, Option, some } from "fp-ts/lib/Option";
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  Form,
+  H1,
+  Icon,
+  Input,
+  Item,
+  Label,
+  Left,
+  Text,
+  View
+} from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import AppHeader from "../../components/ui/AppHeader";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+type State = Readonly<{
+  TransactionCode: Option<string>;
```

Comment:
`lowerCamelCase` for the naming of properties

Reply:
@valentinacala this still needs to be fixed

Reply:
Done!